src/main/java/seedu/duke/ui/Ui.java

```diff
@@ -32,7 +32,7 @@
         "Great! Say, do you think we could find effective power/current consumed by the circuit?"
             + "Or even effective resistance/inductance/capacitance? Yes!\n"
             + "Lets try it out. Type 'calc' followed by 'ceff'/'leff'/'reff'/'power'/'current' and enter. :D",
-        "Great! You're set to use\n" + "CLIrcuit Assistant" + "Enjoy! :)\n type 'exit' to exit this mode.\n"};
+        "Great! You're set to use\n" + "CLIrcuit Assistant" + " Enjoy! :)\n type 'exit' to exit this mode.\n"};
```

Comment:
Any reason why this doesn't use the CLIrcuit Assistant logo? And Enjoy being on the same line? Consider using EXPECTED.txt and ACTUAL.txt, which you'll have to update, to check the formatting.

Reply:
The CLIrcuit Assistant thing, I had removed the logo because of runtest. However, forgot to put it back..